<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dmExport</class>
 <widget class="QWidget" name="dmExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMEx</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="dmLab">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Dummy Target :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="dmBtn">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="agLab">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Agent Export :</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="allRdn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>All</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selRdn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="expBtn">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
